src/main/java/seedu/duke/DeleteCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.duke;
+
+/**
+ * A command that deletes a task.
+ */
+public class DeleteCommand extends Command {
+    public DeleteCommand(String[] words) {
+        super(words);
+    }
+
+    /**
+     * Deletes the task from the list of current tasks.
+     * @param ls The current list of tasks.
+     * @param ui The ui that takes of printing output.
+     * @throws DukeNotSureException If the task being deleted does not exist.
+     */
+    @Override
+    public void execute(TaskList ls, Ui ui) throws DukeNotSureException {
```

Comment:
I like the error handling for the delete command. Maybe the JavaDoc or Exception message can mention the case "task being deleted does not exists" refers to the case where the user types in a number larger than the size of the list.

Reply:
Good idea, thanks.